### DISCUSSION TITLE
Is it possible to disable animations, but enable coloring?

### DISCUSSION THREAD

**Discussion by chrisplim** [Textualize/rich#3568] [Q&A] [answered]
I've tried using `FORCE_COLOR=true` however, this enables the animations, which I don't want in my CI pipeline.
<img width="1419" alt="Screenshot 2024-11-18 at 10 57 33 AM" src="https://github.com/user-attachments/assets/e49d33e4-6fc7-4dd5-b13c-1361b4b7bd8a">

**Comment by kmvanbrunt** [comment, marked as answer]
See the note about disabling animations in CI systems here.
https://rich.readthedocs.io/en/stable/console.html#interactive-mode

**Reply by chrisplim** [reply]
I was confident that I had tried that combination of settings already; but probably mixed the env vars with the console settings.

I got this to work by doing what the [note](https://rich.readthedocs.io/en/stable/console.html#interactive-mode) says

> ... you might want to set force_terminal to True and force_interactive to False.

```
Console(force_terminal=True, force_interactive=False)
```


Thank you for taking the time to point this out!
